src/lms-ds-loader/edfi_lms_ds_loader/Sql_lms_operations.py

```diff
@@ -77,9 +88,8 @@ def disable_staging_natural_key_index(self, table: str) -> None:
 
         assert table.strip() != "", "Argument `table` cannot be whitespace"
 
-        self._exec(
-            f"ALTER INDEX IX_stg_{table}_Natural_Key on lms.stg_{table} DISABLE;"
-        )
+        if self.engine == DbEngine.MSSQL:
+            self._exec(MS_builder.disable_staging_natural_key_index(table))
```

Comment:
Looks like PostgreSQL doesn't have clean functionality for achieving this. Probably need to just drop and recreate the index.

Reply:
https://dba.stackexchange.com/a/116805/183504

Might want to read all of the index definitions, hold them in memory. Drop them. In the "enable" step, just re-execute the create statement. That way the SQL statement that is currently in the middle of a script file doesn't need to be repeated in Python.

Reply:
There seem to be ways to disable indexes, but the general recommendation is to delete them and recreate them later.